packages/web-components/public/switches.ts

```diff
@@ -0,0 +1,25 @@
+import { fillColor, StandardLuminance, SwatchRGB } from '../src/index';
+
+export function toggleBgMode() {
+  const bgChecked = document.getElementById('luminance-switch')!.classList.contains('checked');
```

Comment:
I believe the new value comes as the second argument (prev & next). Using the class name seems dangerous.